catalog-info.yaml

```diff
@@ -17,7 +17,18 @@ spec:
     apiVersion: buildkite.elastic.dev/v1
     kind: Pipeline
     metadata:
-      name: terraform-provider-ec
+      name: terraform-provider-ec-acceptance
     spec:
       repository: elastic/terraform-provider-ec
       pipeline_file: ".buildkite/pipeline.yml"
+      teams:
```

Comment:
The below team perms mirror the current GitHub permissions on the repository - this is also reflected in the GH repository RE : https://github.com/elastic/catalog-info/pull/635